src/main/java/pl/touk/parking/owner/ParkingOwnerService.java

```diff
@@ -0,0 +1,33 @@
+package pl.touk.parking.owner;
```

Comment:
czy to też nie powinno być w domenie?

Reply:
Niekoniecznie. Kluczowe dla mnie było to, żeby serwisy user, operator, owner były w osobnych pakietach (mają różnch aktorów) oraz żeby były razem ze swoimi kontrolerami (są wykorzystwane/zmieniane razem).